bench-vortex/src/bin/clickbench.rs

```diff
@@ -167,26 +167,35 @@ fn main() -> anyhow::Result<()> {
         }
     }
 
-    match args.display_format {
-        DisplayFormat::Table => {
-            for (query_idx, file_format, metric_sets) in metrics {
-                println!("metrics for query={query_idx}, {file_format}:");
-                for (query_idx, metrics_set) in metric_sets.into_iter().enumerate() {
-                    println!("scan[{query_idx}]:");
-                    for metric in metrics_set
-                        .timestamps_removed()
-                        .aggregate()
-                        .sorted_for_display()
-                        .iter()
-                    {
-                        println!("{metric}");
+    match args.engine {
+        Engine::DataFusion => match args.display_format {
+            DisplayFormat::Table => {
```

Comment:
Gonna have to double check how the metrics are set up, but the query measurements run as per usual.

Reply:
metrics are tied to the datafusion vortex exec node, and they get it from the `VortexFile` instances it creates. Probably something for later but it feels like we can plug in to the `EXPLAIN ANALYZE ...` statement in duckdb to provide our custom metrics from VortexFile